packages/cli/src/commands/generate/page/__tests__/page.test.js

```diff
@@ -1,16 +1,56 @@
 global.__dirname = __dirname
+
+let mockFs = false
+let mockFiles = {}
+
+jest.mock('fs', () => {
```

Comment:
I did see https://github.com/redwoodjs/redwood/blob/main/packages/cli/__mocks__/fs.js but I needed more control over when the mocked implementation was used, and when the real implementation was used.

Should I keep this here, as it is, or should I add this to `__mocks__/fs.js`? Or is there some third option I should look in to?

Reply:
I think it's fine to keep this here.